qiskit-superstaq/qiskit_superstaq/superstaq_backend.py

```diff
@@ -422,54 +422,3 @@ def resource_estimate(
         if circuits_is_list:
             return resource_estimates
         return resource_estimates[0]
-
-    def submit_aces(
```

Comment:
why was this removed from `SuperstaqBackend`?

Reply:
I thought we wanted to move it from `SuperstaqBackend` to `SuperstaqProvider` (https://github.com/Infleqtion/client-superstaq/pull/699#issuecomment-1679753907).

Reply:
ohhh im sorry i totally misread that question.

basically i think we want the methods to appear in \*both\* `qss.SuperstaqProvider` and `qss.SuperstaqBackend`. The reason we only had to put them in `qss.SuperstaqBackend` in #687 was that `SuperstaqProvider` was already inheriting them from `gss.Service`

thinking about this a bit more though - it seems like the `qss` version of this is still independent of qiskit? in which case we might not want to remove it from `gss.Service`; instead we can just:
* leave `submit_aces` in `gss.Service` and `qss.SuperstaqBackend`, but update both include the new/updated arguments
* override `submit_aces` in `css.Service` to add support for cirq noise models
* leave `process_aces` alone